OSDOCS-16409: Cluster Admin CQAs (#116629)
openshift/openshift-docs: modules/rosa-adding-tags-cli.adoc

:_mod-docs-content-type: PROCEDURE
[id="rosa-adding-tags-cli_{context}"]
= Adding tags to a machine pool using the {rosa-cli}

[role="_abstract"]

> You can add tags to a machine pool for your {product-title} cluster by using the {rosa-cli-first}. You can not edit the tags after after you create the machine pool.

Reviewer: Doubled word "after after" and "can not" should be "cannot".

```suggestion
You cannot edit the tags after you create the machine pool.
```

--
[source,terminal]
----
$ rosa create machinepool --cluster=<name> --replicas=<replica_count> \
     --name <mp_name> --tags='<key> <value>,<key> <value>'
----

where:

`<key> <value>,<key> <value>`:: Specifies a key and value for each tag, separated by commas.
--
+
.Example output
+
[source,terminal]

> ----

Reviewer: ```suggestion
.Example output
[source,terminal]
----
```
No need for a "+" between a code block and its block title.
